docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,132 @@
+[[configure-endpoint-integration-policy]]
+= Configure an integration policy
```

Comment:
We may want to make this "Configure the endpoint integration policy" since we're specifically referring to endpoint policies (and as the "integration policy" term can apply to any integrations. I know we're in a Security docs section, but I think it could help to be specific.

Reply:
Good idea! I'll change to "Configure the Endpoint Security integration policy".